scripts/rebuilder.py

```diff
@@ -65,27 +64,32 @@ def get_build(self, name: str) -> CoprBuild:
             return None
         return CoprBuild(build)
 
-    def get_regression_info(self, project_owner, project_name):
+    def get_regression_info(
+        self, project_owner: str, project_name: str
+    ) -> dict[str, Any] | None:
         owner_url = project_owner
         if owner_url[0] == "@":
             owner_url = f"g/{owner_url[1:]}"
-        return {
-            "name": self.name,
-            "fail_id": self.latest.id,
-            "url": f"https://copr.fedorainfracloud.org/coprs/{owner_url}/{project_name}/build/{self.latest.id}/",
-            "chroots": self.latest.chroots,
-        }
+        latest = self.latest
+        if latest is not None:
+            return {
+                "name": self.name,
+                "fail_id": latest.id,
+                "url": f"https://copr.fedorainfracloud.org/coprs/{owner_url}/{project_name}/build/{latest.id}/",
+                "chroots": latest.chroots,
+            }
+        return None
 
     @property
-    def latest(self) -> CoprBuild:
+    def latest(self) -> CoprBuild | None:
         return self.get_build("latest")
 
     @property
-    def latest_succeeded(self) -> CoprBuild:
+    def latest_succeeded(self) -> CoprBuild | None:
         return self.get_build("latest_succeeded")
 
 
-def load_tests(loader, tests, ignore):
+def load_tests(loader, tests, ignore):  # type: ignore[no-untyped-def]
```

Comment:
What prevents annotating the types of this function?
Would it be possible to add a comment explaining that?

Reply:
You got me. Fixed.